dev/src/convert.ts

```diff
@@ -182,9 +185,9 @@ function convertValue(fieldValue) {
       };
     case 'arrayValue': {
       const arrayValue: Array<{}> = [];
-      if (is.array(fieldValue.arrayValue.values)) {
-        for (const value of fieldValue.arrayValue.values) {
-          arrayValue.push(convertValue(value));
+      if (is.array(fieldValue.arrayValue!.values)) {
```

Comment:
Would `Array.isArray()` be better (might get rid of one `!` from the next line)?

Reply:
That works, but it doesn't get rid of the `!`. I changed it either way.